.github/workflows/ubuntu.yml

```diff
@@ -752,3 +752,81 @@ jobs:
           ..
         make VERBOSE=1 -j$(nproc)
         sudo make install
+
+  gcc-10-AVC-HEVC-exclude:
+    runs-on: ubuntu-20.04
+    env:
+      CC: /usr/bin/gcc-10
+      CXX: /usr/bin/g++-10
+      ASM: /usr/bin/gcc-10
+    steps:
+    - name: checkout media-driver
+      uses: actions/checkout@v2
+      with:
+        path: media
+    - name: checkout libva
+      uses: actions/checkout@v2
+      with:
+        repository: intel/libva
+        path: libva
+    - name: checkout gmmlib
+      uses: actions/checkout@v2
+      with:
+        repository: intel/gmmlib
+        path: gmmlib
+    - name: install prerequisites
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends \
+          cmake \
+          libdrm-dev \
+          libegl1-mesa-dev \
+          libgl1-mesa-dev \
+          libx11-dev \
+          libx11-xcb-dev \
+          libxcb-dri3-dev \
+          libxext-dev \
+          libxfixes-dev \
+          libwayland-dev \
+          make
+    - name: print tools versions
+      run: |
+        cmake --version
+        $CC --version
+        $CXX --version
+    - name: build libva
+      run: |
+        cd libva
+        ./autogen.sh --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu
+        make -j$(nproc)
+        sudo make install
+    - name: build gmmlib
+      run: |
+        cd gmmlib
+        mkdir build && cd build
+        cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=/usr/lib/x86_64-linux-gnu ..
+        make VERBOSE=1 -j$(nproc)
+        sudo make install
+    - name: build media-driver
+      run: |
+        cd media
+        mkdir build && cd build
+        cmake -DENABLE_NONFREE_KERNELS=OFF \
+          -DENABLE_KERNELS:BOOL=OFF \
+          -DBUILD_KERNELS:BOOL=OFF \
```

Comment:
it means, it only enable the HW feature without media kernel support
even vp8 DP is excluded?

Reply:
Updated to more feature build